server/src/test/java/com/saralein/server/middleware/FileMiddlewareTest.java

```diff
@@ -0,0 +1,50 @@
+package com.saralein.server.middleware;
+
+import com.saralein.server.FileHelper;
+import com.saralein.server.mocks.MockCallable;
+import com.saralein.server.mocks.MockHandler;
+import com.saralein.server.request.Request;
+import org.junit.Before;
+import org.junit.Test;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import static org.junit.Assert.assertFalse;
+
+public class FileMiddlewareTest {
+    private MockHandler mockHandler;
+    private MockCallable mockCallable;
+    private Middleware fileMiddleware;
+
+    @Before
+    public void setUp() {
+        Path root = Paths.get(System.getProperty("user.dir"), "src/test/public");
+        mockHandler = new MockHandler(200, "File response");
+        mockCallable = new MockCallable();
+        fileMiddleware = new FileMiddleware(new FileHelper(root), mockHandler).apply(mockCallable);
+    }
+
+    @Test
+    public void callsHandlerForFiles() {
```

Comment:
These test names are based on the implementation of the SUT. It would be nice to have names that are more meaningful and put in terms of the domain, like `itHandlesTheFile` and `itForwardsToTheNextMiddleware`. Does that make sense?

Reply:
Yes, makes total sense.